steps_file.js

```diff
@@ -1,10 +1,61 @@
-// in this file you can append custom step methods to 'I' object
+const { continueButton } = require("./pages/account");
 
+    emailField = {css: "#input-email"};
+    passwordField = {css: "#input-password"};
+    signInButton = { xpath: '//a[text()="Sign In"]'};
+    loginButton = { xpath: '//input[@type="submit"]'};
+    myOrdersText = { xpath: '//h2[text()="My Orders"]'};
+    addToCartButton = { xpath: '//button[@id="button-cart"]'};
+    CartButton = { xpath: '//div[@id="cart"]'};
+    checkoutButton = {xpath: '//a[@class="btn-primary btn-r"]'};
+    firstNameField = {xpath: '//div/input[@name="firstname"]'};
+    lastNamefield = {xpath: '//div/input[@name="lastname"]'};
+    address1Field = {xpath: '//div/input[@name="address_1"]'};
+    cityField = {xpath: '//div/input[@name="city"]'};
+    postCodeField = {xpath: '//div/input[@name="postcode"]'};
+    countryField = {xpath : '//div[@id="sbHolder_44325861"]/a[2]'};
+    regionStateField = {xpath : '//div[@id="sbHolder_26577242"]/a[2]'};
+    selectRegionField = {xpath : '//ul[@id="sbOptions_51230567"]/li[8]/a'};
+    continueButton = {xpath : '//input[@id="button-payment-address"]'};
```

Comment:
remove this continueButton or the one in 1st row. you are trying to reassign value for constant variable.